zebra/zebra_vty.c

```diff
@@ -458,6 +460,9 @@ static void zebra_show_ip_route_opaque(struct vty *vty, struct route_entry *re,
 			if (bzo.lcommunity[0] != '\0')
 				vty_out(vty, "    Large-Communities: %s\n",
 					bzo.lcommunity);
+
+			vty_out(vty, "    Best path reason : %s\n",
```

Comment:
"Selection reason" instead of Best path reason, to make this consistent with what we already have in BGP ?

Reply:
Fixed.